src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +54,51 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService
+                = (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        User bob = new User();
+        bob.setEmail("123@123");
+        bob.setPassword("qwerty");
+
+        AuthenticationService auth
+                = (AuthenticationService) injector.getInstance(AuthenticationService.class);
+        try {
+            User register = auth.register(bob.getEmail(), bob.getPassword());
+            System.out.println(register);
+        } catch (RegistrationException e) {
+            throw new RuntimeException(e);
+        }
+        try {
+            User login = auth.login(bob.getEmail(), bob.getPassword());
+            System.out.println(login);
+        } catch (AuthenticationException e) {
+            throw new RuntimeException(e);
+        }
+
+        User sameEmailNotOk = new User();
+        sameEmailNotOk.setEmail("123@123");
+        sameEmailNotOk.setPassword("123");
+        try {
+            auth.register(sameEmailNotOk.getEmail(), sameEmailNotOk.getPassword());
+        } catch (RegistrationException e) {
+            System.out.println(e);
+        } finally {
+            User loginDoesNotExistNotOk = new User();
+            loginDoesNotExistNotOk.setEmail("qwe@qwe");
+            loginDoesNotExistNotOk.setPassword("qwe");
+            try {
+                auth.login(loginDoesNotExistNotOk.getEmail(),
+                        loginDoesNotExistNotOk.getPassword());
+            } catch (AuthenticationException e) {
+                throw new RuntimeException(e);
+            }
+        }
```

Comment:
It's not a good idea to use a finally block for this purpose
Let’s  move this code outside